src/components/message/Message.tsx

```diff
@@ -0,0 +1,51 @@
+import { FC } from 'react'
+import { UserInterface } from '~/types'
+import { Box, Avatar, IconButton, Typography } from '@mui/material'
+import { Favorite, Flag, MoreHoriz } from '@mui/icons-material'
+import { styles } from '~/components/message/Message.styles'
+
+import { spliceSx } from '~/utils/helper-functions'
+
+interface MessageProps {
+  user: UserInterface
+  timestamp?: Date | string
+  messageContent: string
+}
+
+const Message: FC<MessageProps> = ({ user, timestamp, messageContent }) => {
+  return (
+    <Box sx={styles.root}>
+      <Box sx={styles.messageBox}>
+        <Avatar src={user.photo} sx={styles.avatar} />
```

Comment:
use UserProfileInfo for photo , name, and time

Reply:
> use UserProfileInfo for photo , name, and time

seems like this is a better idea, used it 👍